tmdrv_devices/thrustmaster_tmx.py

```diff
@@ -0,0 +1,9 @@
+name = 'Thrustmaster TMX'
+idVendor = 0x044f
+idProduct = [ 0xb65d, 0xb67e]
+control = [
+	{'step':1, 'request_type':0x41, 'request':83, 'value':0x0001, 'index':0x0000, 'data':b''},
+	{'step':2, 'request_type':0x41, 'request':83, 'value':0x0004, 'index':0x0000, 'data':b''},
```

Comment:
`value` here looks like it should be `0x0007`, according to packet number 653 in `Thrustmaster_TMX_2017_07_22(Replug Only).pcap`.